### ISSUE TITLE
error namespace 

### ISSUE THREAD

**Issue by matraux** [axima/php-payment-gate#3]
File `axima\payment-gate\src\Utils.php` have bad namespace. There is `namespace Axima\PaymentGate;` and should be `namespace Pays\PaymentGate;` or composer file should be changed from `Pays\PaymentGate` to `Axima\PaymentGate`.

**Comment by greeny** [comment]
This was already fixed in 8ec3bda with #2

**Comment by matraux** [comment]
> This was already fixed in [8ec3bda](https://github.com/axima/php-payment-gate/commit/8ec3bda4a1f129682705994845e5795fb568d748) with #2

Try to download package with composer. Look at the screenshot. Probably there will be old public package.

![obrazek](https://github.com/axima/php-payment-gate/assets/46919973/e87fa90e-8589-4f41-b188-865eb4028d99)

Composer config:
```
{
    "require": {
        "axima/payment-gate": "^1.0"
    }
}
```
